src/components/Image/Avatar.js

```diff
@@ -1,36 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { variant } from 'styled-system';
 
 import { Box } from '../Box';
 import Tile from './Tile';
 
-const SIZES = {
-  small: '24px',
-  icon: '32px',
-  base: '96px',
-};
-
 const Wrapper = styled(Box)`
-  width: ${({ size }) => (SIZES[size] ? SIZES[size] : SIZES['base'])};
-  height: ${({ size }) => (SIZES[size] ? SIZES[size] : SIZES['base'])};
+  ${variant({
```

Comment:
if you'd like to keep `ratio` as the prop, check this:
https://styled-system.com/variants#custom-prop-name

basically add `prop: 'ratio',` above line 11. Keeps it easy to read.

Reply:
`size` in this case,` ratio` below.

Reply:
Love this.... Fixing now